### PR TITLE
Dom menu

### PR DESCRIPTION
**Pull request by Pazzo97**
Added DOM to portfolio project

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -651,3 +597,119 @@ textarea {
...
+  }
+
+  #envelope {
+    display: none;
+  }
+
+  .navLink {
+    font-size: 40px !important;
+  }
+
```
Comment (thecodechaser) [inline style.css]: - [ ] Good job on the design but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2925) design as much as possible. Please reduce the top margin of your mobile menu, Use the correct font size, and font-weight and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: style.css [hunk 2]
```diff
@@ -651,3 +597,119 @@ textarea {
...
+    margin-right: 30px;
+  }
+
+  nav .mainMenu .closeMenu {
+    display: block;
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    margin-top: 25px;
+  }
```
Comment (thecodechaser) [inline style.css:679]: - [ ] Please use the correct size for your close(X) button, and make sure it matches the design on the template.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Pazzo97 and Team👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] This branch has conflicts with the base branch, Please fix the conflicts. This can help you [Resolve GitHub conflicts ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github).

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @kelvinbush   @Pazzo97 ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- When the user clicks the hamburger button, the mobile menu appears.✔️ 
- When the user clicks the close button, the mobile menu disappear.✔️ 
- When the user clicks any of the mobile menu options, a correct part of the page is displayed.✔️ 
- When the user clicks any option of the mobile menu, the mobile menu disappear.✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


- [OPTIONAL ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)



Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**